ambari-server/src/main/java/org/apache/ambari/server/security/encryption/MasterKeyServiceImpl.java

```diff
@@ -302,11 +275,7 @@ private void initializeFromFile(File masterFile) throws Exception {
       List<String> lines = FileUtils.readLines(masterFile, "UTF8");
       String tag = lines.get(0);
       LOG.info("Loading from persistent master: " + tag);
-      String line = new String(Base64.decodeBase64(lines.get(1)));
```

Comment:
do we have a string that is encoded  twice into Base64?

Reply:
yes; this is how it's working for a long time...I kept it as is to support backward compatibility